fixtures/error-types/tests/bindings/test.kts

```diff
@@ -0,0 +1,113 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import uniffi.error_types.*;
+import kotlinx.coroutines.*
+
+try {
```

Comment:
Are these outputs from stock uniffi, not uniffi-bindgen?

Reply:
Yes, they're left in error. Removed.